apps/nextjs/src/app/onboarding/page.tsx

```diff
@@ -1,5 +1,5 @@
 import OnBoarding from "./onboarding";
 
-export default function OnBoardingPage() {
+export default async function OnBoardingPage() {
```

Comment:
Does this need an explicit "use server" so it's clear it's a server component?

Reply:
Ah, I was originally doing the demo metadata upgrade in the server component here but we didn't really have the precedent for making calls there. I'll remove the async declaration and leave the file as-is